packages/react/src/components/uiComponents.tsx

```diff
@@ -274,3 +275,14 @@ export const OrganizationList = withClerk(({ clerk, ...props }: WithClerkProp<Or
     />
   );
 }, 'OrganizationList');
+
+export const __experimental_GoogleOneTap = withClerk(({ clerk, ...props }: WithClerkProp<OneTapProps>) => {
```

Comment:
❓ Are we sure a component name starting with `_` works well in JSX?

Reply:
Using it with our playground did not give me trouble